starters/features/partytown/src/components/partytown/partytown.unit.tsx

```diff
@@ -0,0 +1,19 @@
+import { ElementFixture } from '@builder.io/qwik/testing';
```

Comment:
This file will be added to developers starter, please remove.

Also, they will not have this deps installed